zds/featured/models.py

```diff
@@ -30,6 +39,15 @@ def __unicode__(self):
 
 
 class FeaturedMessage(models.Model):
+    """
+        The Featured Message is the one-line information display below the navbar on the home page
+        This message is divided in three parts:
+            - The hook : display in bold, show the topic of the message (i.e.: "New", "Warning", "Info", ...)
+            - The message : the info message itself (i.e.: "The site will be down for maintenance tomorow")
+            - The "tell me more" url : A tell me more button linking to the url of the ressource
```

Comment:
"ressource" est vague, je dirais plutôt "of a page explaining the message with more details"